src/main/java/net/minecraftforge/fml/common/registry/VillagerRegistry.java

```diff
@@ -160,7 +162,6 @@ public void registerVillageCreationHandler(IVillageCreationHandler handler)
      * @param defaultSkin
      */
     @SideOnly(Side.CLIENT)
-    @Deprecated // Doesn't work at all.
```

Comment:
it is currently used by the renderer to get the texture, should I also make a patch for that to use the one from the profession?